src/main/java/nourl/mythicmetals/data/MythicTags.java

```diff
@@ -0,0 +1,246 @@
+package nourl.mythicmetals.data;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemConvertible;
+import net.minecraft.tag.TagKey;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.util.registry.RegistryKey;
+import nourl.mythicmetals.MythicMetals;
+import nourl.mythicmetals.blocks.BlockSet;
+import nourl.mythicmetals.blocks.MythicBlocks;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+
+@SuppressWarnings("unused")
+public class MythicTags {
```

Comment:
This should be `MythicTagHelpers`, to prevent confusion with the badly named `RegisterTags` in the registry package. I want to refactor that after this to MythicTags

Reply:
I really think they should be folded into this class as it makes more sense to keep tags in one area and not many

Reply:
Should this class ever be initialized when they are not used for datagen?

Reply:
Fixed